docs/index.md

```diff
@@ -115,12 +115,17 @@ Travis will then kick off a build against the new tag created and deploy that bu
 
 There are a few things that can be checked if you review a pull request against one of our modules:
 
-* Correct email address used in the commits (if so, travis displays the avatar next to the commit)?
+* Correct email address used in the commits (if so, github displays the avatar next to the commit)?
 * Is this a bugfix, modulesync, breaking change, enhancement, docs update? Label it with `bug`, `modulesync`, `backwards-incompatible`, `enhancement`, `docs`
 * Are updates to the README.md needed but missing? Label it with `needs-docs`
+* Has the file documented params or examples in the header? This needs to be updated as well
```

Comment:
Should we start using https://github.com/voxpupuli/puppet-lint-param-docs? It needs a whitelist of files to check (like https://github.com/theforeman/puppet-foreman/blob/645a48c15dbf30901179afb198d4ab392484335d/Rakefile#L21-L24)

Reply:
I would like to enforce puppet-strings docs for new params, but most modules don't have existing docs :(. I'm not sure how helpful the linter plugin would be?